### DISCUSSION TITLE
organizzazione fisica - 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#81] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/20f12389-4c38-4b48-a855-b09d1079a3a6)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-23 162437](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/103a8cf0-b925-455c-8a30-8e91aaa592b1)
![Immagine 2024-01-23 162452](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/f18b00dc-ea8e-4ff6-bdfb-43184d7a9681)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/47409278-485f-4c8c-9369-1871a711a4eb)

**Reply by 7Andrei** [reply]
Hai sbagliato ad arrotondare il RpB che viene 15 non 16, quindi anche il BxFP e il BxFI sono sbagliati
